tests/unit/common/buildApp.spec.ts

```diff
@@ -1,25 +1,47 @@
-import TermboxRequest from '@/common/TermboxRequest';
 import buildApp from '@/common/buildApp';
-import FingerprintableEntity from '@/datamodel/FingerprintableEntity';
+import App from '@/components/App.vue';
+
+const mockGetChildComponents = jest.fn();
+jest.mock( '@/common/getChildComponents', () => ( {
+	__esModule: true,
+	default: ( app: any ) => mockGetChildComponents( app ),
+} ) );
+
+jest.mock( '@/store', () => ( {
+	__esModule: true,
+	default: {},
```

Comment:
Witchcraft, took me a while. The value of this here should actually be the same object also passed to `mockAsyncData1` & `mockAsyncData2`'s first parameter, right?

Reply:
Yes, I'll create a variable for it so it's more obvious.